### PR TITLE
Update Applied Filters

### PR DESCRIPTION
**Pull request by kenjinp**
useState to update filters

put stuff into a `Workspace` functional component because hooks wouldn't work inside the async/await main() function

### CODE REVIEW

File: src/components/FiltersList/index.tsx [hunk 1]
```diff
@@ -10,18 +10,22 @@ export interface Filter {
   }[];
 }
 
-export interface FacetUpdatePayload {
+export interface AppliedFilters {
+  [filterName: string]: string[];
+}
+
+export interface FilterUpdatePayload {
   facetName: string;
```
Comment (julienmachon) [inline src/components/FiltersList/index.tsx]: maybe this should be
```suggestion
  filterName: string;
```

File: src/components/FiltersList/index.tsx [hunk 2]
```diff
@@ -10,18 +10,22 @@ export interface Filter {
...
-  appliedFilters?: { [filterName: string]: string[] };
+interface FilterProps {
+  appliedFilters?: AppliedFilters;
   filters?: Filter[];
-  updateFilters?: (filterUpdate: FacetUpdatePayload) => void;
+  updateFilters?: (filterUpdate: FilterUpdatePayload) => void;
 }
 
-const FacetList: React.FunctionComponent<FacetProps> = props => {
+const FacetList: React.FunctionComponent<FilterProps> = props => {
```
Comment (julienmachon) [inline src/components/FiltersList/index.tsx]: Maybe this should be
```suggestion
const FilterList: React.FunctionComponent<FilterProps> = props => {
```

File: src/index.tsx [hunk 3]
```diff
@@ -4,8 +4,7 @@ import { NexusClient, createNexusClient } from "@bbp/nexus-sdk";
 import { NexusProvider } from "@bbp/react-nexus";
 import { setUpSession, setToken } from "./utils/auth";
 import Header from "./components/Header";
-import Filters from "./containers/Filters";
-import Results from "./components/Results";
+import Workspace from "./containers/Workspace";
```
Comment (julienmachon) [inline src/index.tsx]: can you move this into a `./SparqlView.tsx` file instead? Worspace doesn't mean much to me

**Comment by kenjinp** [comment]
Again codesandbox broke things

**Pull request closed without merging**